server/static/jsx/scholarship.jsx

```diff
@@ -0,0 +1,96 @@
+/** @jsx React.DOM */
+define(['ext/jquery', 'ext/react'], function($, React) {
+  var ScholarshipContainer = React.createClass({
```

Comment:
Can you set `propTypes` here, please? http://facebook.github.io/react/docs/reusable-components.html#prop-validation

It acts as really helpful documentation and will complain in dev if the types don't match up